viamonvif/device/device.go

```diff
@@ -88,11 +91,16 @@ type GetCapabilities struct {
 
 // NewDevice function construct a ONVIF Device entity.
 func NewDevice(params Params, logger logging.Logger) (*Device, error) {
+	if params.Context == nil {
```

Comment:
This function should take a context and it should be required.

Reply:
Done